components/I18nFormatters.js

```diff
@@ -9,5 +9,24 @@ export const I18nSupportLink = chunks => (
   <StyledLink href="mailto:support@opencollective.com">{chunks.length || 'support@opencollective.com'}</StyledLink>
 );
 
-const I18nFormatters = { strong: I18nBold, i: I18nItalic, SupportLink: I18nSupportLink };
+export const I18nSignLink = msg => <StyledLink href="/signin">{msg}</StyledLink>;
+export const I18nTOSLink = msg => (
+  <StyledLink color="black.500" href="/tos">
+    {msg}
+  </StyledLink>
+);
+export const I18nPrivacyLink = msg => (
+  <StyledLink color="black.500" href="/privacypolicy">
```

Comment:
Same here, should use `Link`. Also we should not add any styles here, this library is solely about i18n and should not interfere with the styles.

Reply:
done :rocket: